akka-stream-tests/src/test/scala/akka/stream/io/FileSinkSpec.scala

```diff
@@ -52,6 +53,17 @@ class FileSinkSpec extends AkkaSpec(UnboundedMailboxConfig) {
       }
     }
 
+    "create new file if not exists" in assertAllStagesStopped {
+      targetFile({ f ⇒
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Non ascii characters are not allowed](https://www.codacy.com/app/vasseurjb/akka/file/2046285078/issues/source?bid=3132306&fileBranchId=3132378#l57)